.doks/doks-cluster/Makefile

```diff
@@ -4,13 +4,15 @@ TF_EXTRA_OPTS ?= TF_IN_AUTOMATION=1 AWS_SDK_LOAD_CONFIG=1 TF_CLI_CONFIG_FILE=$(T
 TF_PARALLELISM ?= 30
 TF ?= $(TF_EXTRA_OPTS) terraform
 WORKSPACE ?= ${DO_TF_WORKSPACE}
+NAMESPACE?=codacy
 VARS_FILE ?= deploy.tfvars
 
 # variables
-K8S_VERSION ?= 1.15
+K8S_VERSION ?= 1.14
```

Comment:
are we going back to 1.14?

Reply:
this is the k8s version that was originally on the cluster and we were actually passing this value onto the makefile.
:)
if we want to move to to 1.15 then we need to create a new cluster :-(

Reply:
Also, by specifying this version here, with the terraform apply --auto-approve that we have below, we will guarantee that the cluster is up to date with regards to patch versions of k8s 1.14 automatically